requirements.txt

```diff
@@ -1,3 +1,5 @@
 requests
```

Comment:
Is `requests` still needed explictly?

Reply:
Yes, for the `/swagger.json` endpoint

Reply:
Ah, thanks. Looks like it is being used in the bdbag code, too.